Description: Fix FTBFS with CMake 4
Author: Adrian Bunk <bunk@debian.org>
Bug-Debian: https://bugs.debian.org/1113567

--- terminus-3.6.0.orig/CMakeLists.txt
+++ terminus-3.6.0/CMakeLists.txt
@@ -2,8 +2,7 @@
 ### Do not edit
 
 project (terminus)
-cmake_minimum_required (VERSION 2.6)
-cmake_policy (VERSION 2.8)
+cmake_minimum_required (VERSION 3.5)
 list (APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 enable_testing ()
 option(ICON_UPDATE "Update the icon cache after installing" ON)
